charts/hono/values.yaml

```diff
@@ -1823,12 +1905,21 @@ jaegerBackendExample:
   # allInOneImage contains the name (including tag)
   # of the container image to use for the example Jaeger back end.
   allInOneImage: "jaegertracing/all-in-one:1.31"
-  # livenessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
+  # [DEPRECATED: use probes instead] livenessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
   # configuration property of the component's liveness probe.
+  # The value of the top level "livenessProbeInitialDelaySeconds" property will be used if not set.
   livenessProbeInitialDelaySeconds: 300
-  # readinessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
+  # [DEPRECATED: use probes instead] readinessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
   # configuration property of the component's readiness probe.
+  # The value of the top level "readinessProbeInitialDelaySeconds" property will be used if not set.
   readinessProbeInitialDelaySeconds: 10
+  probes:
+    livenessProbe:
+      httpGet:
+        path: "/"
+    readinessProbe:
+      httpGet:
+        path: "/"
```

Comment:
IMHO we should set the initialDelaySeconds in the probes here and leave the deprecated properties empty. That way we also provide an example of how to set the values in the desired way ...